library/oneview_enclosure_group.py

```diff
@@ -100,10 +100,10 @@
     type: dict
 '''
 
-from ansible.module_utils.oneview import OneViewModuleBase
+from ansible.module_utils.oneview import OneViewModule
 
 
-class EnclosureGroupModule(OneViewModuleBase):
+class EnclosureGroupModule(OneViewModule):
     MSG_CREATED = 'Enclosure Group created successfully.'
     MSG_UPDATED = 'Enclosure Group updated successfully.'
     MSG_DELETED = 'Enclosure Group deleted successfully.'
```

Comment:
These constants are not used in the file

Reply:
These messages are used by resource_present and resource_absent methods defined in the base class. All these class variables are defined in the base class with a generic message for create, delete, update, etc. Here we are overwriting those messages to get the resource-specific messages.

Reply:
Got it.